### ISSUE TITLE
Composed statuses are not shown in "Racks" and "Jobsmap" view

### ISSUE THREAD

**Issue by mehdid** [rackslab/Slurm-web#144]
The interface only shows "available" and "reserved" states. Other states are not considered somehow.

**Comment by rezib** [comment]
The node state string returned by PySLURM[1] is:

```
slurm_node_state_string[+CLOUD][+COMPLETING][+DRAIN|+FAIL][+POWER]
```

The values returned by `slurm_node_state_string()` [1] are:

* `MAINT`
* `DRAINING`
* `ERROR`
* `DRAINED`
* `FAILING`
* `FAIL`
* `POWER_DOWN`
* `POWER_UP`
* `DOWN`
* `ALLOCATED`
* `COMPLETING`
* `IDLE`
* `PERFCTRS`
* `RESERVED`
* `MIXED`
* `FUTURE`
* `RESUME`
* `UNKNOWN`

My guess here is that the `[+COMPLETING][+DRAIN|+FAIL][+POWER]` added by PySLURM are probably wrong. They are arguably bugs in PySLURM. Even though the CLOUD tag is managed like a state inside Slurm data structures, the `+CLOUD` tag should probably be managed outside of the node state string, typically with an additional boolean inside the node hash. It is not really a state since it is a static attribute that does not change over node lifetime.

My point is that we should discuss with PySLURM developers to removed those composed multi-values string states. They are arguably wrong and too hard to parse. Then, we should properly define both the state color and the node color for each of them in the above list. This list should appear in the documentation and in the legend of the racks and jobmaps views as well.

[1] https://github.com/PySlurm/pyslurm/blob/master/pyslurm/pyslurm.pyx#L2726
[2] https://github.com/SchedMD/slurm/blob/master/src/common/slurm_protocol_defs.c#L2485

**Comment by rezib** [comment]
This issue concerns Slurm-web v2 which is not maintained anymore. You are highly encouraged to test the new version v3.0.0 that is not impacted by this issue. The quick start guide for v3.0.0 is available online: https://docs.rackslab.io/slurm-web/install/quickstart.html

Unless someone is motivated to maintain the old version of Slurm-web or you have a justified reason to keep this issue open, it will be closed in a few weeks.

**Comment by rezib** [comment]
For the reasons explained in the previous comment, I finally close this issue.
